packages/node_modules/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -84,22 +84,20 @@ const Authorization = WebexPlugin.extend({
     if (location.query.state) {
       location.query.state = JSON.parse(base64.decode(location.query.state));
     }
+    else {
+      location.query.state = {};
+    }
 
+    const {email} = location.query.state;
 
     this._verifySecurityToken(location.query);
     this._cleanUrl(location);
 
     // Wait until nextTick in case `credentials` hasn't initialized yet
     process.nextTick(() => {
-      // if a particular CI was included in the state, use that
-      if (location.query.state && location.query.state.idbroker) {
-        this.config.idbroker.url = location.query.state.idbroker;
-      }
-      if (location.query.state && location.query.state.identity) {
-        this.config.identity.url = location.query.state.identity;
-      }
-
-      this.requestAuthorizationCodeGrant({code})
+      this.webex.internal.services.collectPreauthCatalog({email})
+        .catch(() => Promise.resolve())
```

Comment:
Could you add a note about why we are ignoring errors from `collectPreauthCatalog`?

Reply:
This is a fallback is to prevent errors when an encoded email value is not passed into the method. It should be for legacy fallbacks.